src/api.c

```diff
@@ -254,10 +251,6 @@ void amy_start(amy_config_t c) {
     amy_profiles_init();
     sequencer_start();
     oscs_init();
-    if(amy_global.config.set_default_synth) {
-	amy_default_setup();
-	if (amy_global.config.startup_bleep) bleep_synth(0);
-    } else {
-	if (amy_global.config.startup_bleep) bleep(0);
-    }
+    if(AMY_HAS_DEFAULT_SYNTHS)amy_default_setup();
+    if(AMY_HAS_STARTUP_BLEEP) bleep_synth(0);
```

Comment:
No, that's not the logic.  `bleep_synth` can only be used if `HAS_SYNTH`, otherwise we have to use `bleep`.  But if you have `SYNTH` you shouldn't use `bleep` (because it scribbles on an osc without regard to any `SYNTH` setup).  Actually, I changed `bleep` to use `osc = AMY_OSCS - 1`, so maybe it's OK to use in both cases.

Reply:
Not sure what the fix is here if any?